bower.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "polymerfire",
-  "version": "0.9.4",
+  "name": "polymerfire-extended",
+  "version": "0.9.5",
   "authors": [
```

Comment:
I don't think you can change the name of the project or version. That will be done by person responsible for maintaining this project.